sdk/resourcemanagerhybrid/README.md

```diff
@@ -60,9 +56,8 @@ The services available via `azure-resourcemanager` are listed as below:
 - Network
 - Resources
 - Storage
-</details>
 
-In the case where you are interested in certain service above or the service not included in the multi-service package, you can choose to use the single-service package for each service. Those packages follow the same naming patterns and design principals. For example, the package for Media Services has the following artifact information.
+In the case where you are interested in certain service above, the service not included in the multi-service package, or an Azure-only service, you can choose to use the single-service package for each service. Those packages follow the same naming patterns and design principals. For example, the package for Media Services has the following artifact information.
 
 [//]: # ({x-version-update-start;com.azure.resourcemanager:azure-resourcemanager-mediaservices;dependency})
 ```xml
```

Comment:
You probably want to remove this section. As this points to other 100+ packages, which most of them will not work for Azure Stack.

Reply:
Changed this to use show how to use Compute as a single package instead, left off the Azure-specific things